### ISSUE TITLE
Kill completion doesn't work

### ISSUE THREAD

**Issue by me-and** [me-and/Cygwin-fzf#1]
Using fzf installed from the upstream repository, kill completion produces a list that looks a lot like ps output.

Using fzf installed from this repository, tapping <kbd>Tab</kbd> just produces a terminal bell, and tapping a second time produces output that looks something like the below, with no interactive elements:

```
$ kill
/usr/bin/bash       1300                COMMAND
/usr/bin/mintty     15:53:45            PGID
/usr/bin/ps         16:08:51            PID
/usr/bin/ssh-agent  16:10:27            PPID
?                   2948                pty0
1                   3636                STIME
10048               8048                TTY
10224               8184                UID
1063214             9816                WINPID
```

**Comment by me-and** [comment]
Ah, this is being overwritten by the `util-linux` bash-completion script, which is being run after the `fzf` one. A thing to ponder: how to make sure that fzf's completion options aren't overridden by others?

The obvious fix would be to rename the completion script to `zzzzz-fzf`, since the scripts are run in alphabetical order, but that feels incredibly hacky, not least because I don't think it's safe to assume that alphabetical order will be maintained.

**Comment by me-and** [comment]
Fixed by 167c37ed5d.
